packages/commonwealth/client/scripts/views/pages/finish_social_login.tsx

```diff
@@ -4,14 +4,17 @@ import { PageLoading } from 'views/pages/loading';
 import ErrorPage from 'views/pages/error';
 import { handleSocialLoginCallback } from 'controllers/app/login';
 import { initAppState } from 'state';
-import app from 'state';
 
 const validate = async (setRoute) => {
+  const params = new URLSearchParams(window.location.search);
+  const chain = params.get('chain');
+
   try {
     await handleSocialLoginCallback();
     await initAppState();
-    if (app.activeChainId()) {
-      setRoute(`/account/${app.activeChainId()}`);
+
+    if (chain) {
+      setRoute(`/${chain}`);
     } else {
       setRoute('/dashboard');
```

Comment:
Can we change this dashboard redirect to just go to `/`, so that we don't need to worry about custom domain? (if on a custom domain, it'll go to discussions or overview; if not, it'll go to dashboard).